code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -26,6 +28,8 @@
 	var/wrapped = 0 //Is the food wrapped (preventing one from eating until unwrapped)
 	var/dried_type = null //What can we dry the food into
 	var/deepfried = 0 //Is the food deep-fried ?
+	var/rotten = 0 //Is the food rotten ?
+	var/cold = 50 //How much time until it starts decomposing. Timer stops if inside a cold environment
```

Comment:
Given our tickrate, I believe 50 ticks is less than a minute

Reply:
One tick is two seconds if memory serves.

Reply:
We really should have a coder wiki page where we have all this information easily accessible

If I recall correctly, the powernet runs on a different tickrate too, not that it's relevant here but just as an example.